src/components/standalone/security/ips/IpsFilterBypass.vue

```diff
@@ -60,18 +81,43 @@ const onSort = (payload: any) => {
   sortKey.value = payload.key
   sortDescending.value = payload.descending
 }
+
+function savedBypass(bypass: ByPass) {
+  byPasses.value.push(bypass)
```

Comment:
We usually re-trigger the read api after a write operation, to ensure frontend/backend consistency